src/malli/time.clj

```diff
@@ -0,0 +1,69 @@
+(ns malli.time
+  (:require
+   [clojure.string :as str]
+   [malli.core :as m])
+  (:import
+   (java.time Duration LocalDate OffsetDateTime OffsetTime)))
+
+(defn ->safe-parser
+  [f]
+  (fn -parse [x]
+    (if (string? x)
+      (try
+        (f x)
+        (catch Exception _ x))
+      x)))
+
+(defn ->error-reporter
+  [parser message]
+  (fn -report [value]
```

Comment:
Should there be a second param here?
If I specify a property as `:local-date` and try to send a non-parsable value, e.g. "2022-01-aa", I get:
_Error: Wrong number of args (2) passed to: slakt-service.malli.time/->error-reporter/-report--28426_ 

If I change this line to 
`(fn -report [value _]`
I get the error message "Should be local-date or LocalDate" as expected